elixir/apps/domain/test/domain/accounts_test.exs

```diff
@@ -127,4 +127,52 @@ defmodule Domain.AccountsTest do
       assert ensure_has_access_to(subject, account) == {:error, :unauthorized}
     end
   end
+
+  describe "create_account/1" do
+    test "creates account given a valid name" do
+      assert {:ok, account} = create_account(%{name: "foo"})
+      assert account.name == "foo"
+    end
+
+    test "creates account given a valid name and valid slug" do
```

Comment:
We also need to test what happens when a name or slug is already taken.

Reply:
Ah, good catch, not sure how I missed that one.

Reply:
Tests have been updated